handbook/business-operations/vendor-questionnaires.md

```diff
@@ -27,14 +35,19 @@
 ## Governance and risk management
 | Question | Answer                                                                                                                                                 |
 | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Does Fleet have documented information security baselines for every component of the infrastructure (e.g., hypervisors, operating systems, routers, DNS servers, etc.)?  | YWe follow best practices for the given system. For instance, with AWS we utilize AWS best practices for security including GuardDuty, CloudTrail, etc.                                                                |
+| Does Fleet have documented information security baselines for every component of the infrastructure (e.g., hypervisors, operating systems, routers, DNS servers, etc.)?  | Fleet follows best practices for the given system. For instance, with AWS we utilize AWS best practices for security including GuardDuty, CloudTrail, etc.                                                                |
+
+## [Business continuity](https://fleetdm.com/handbook/business-operations/security-policies#business-continuity-plan)
+| Question | Answer                                                                                                                                                 |
+| ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Please provide your application/solution disaster recovery RTO/RPO | RTO and RPO intervals differ depending on the service that is impacted. Please refer to https://fleetdm.com/handbook/business-operations/security-policies#business-continuity-and-disaster-recovery-policy                                                               |
```

Comment:
In keeping with the cadence of the handbook as a whole, any time we light up headings it is to link TO that particular section. In this case, we are linking FROM. 

Is there any way we could do something matching what you have done here? or even a section of the table that just states related docs